src/dp5/params.jl

```diff
@@ -0,0 +1,31 @@
+# define constants
```

Comment:
is this generated by some formula? maybe just eval that at precompile time?

Reply:
dp5, maybe, but for dp8, the issue is that these are optimized parameters.

I can make them evaluated at precompile time. We haven't done much in terms of optimization at this stage just trying to reorganize the code and make sure its type stable at the very least.